### PR TITLE
Remove Yarn

### PR DESCRIPTION
**Pull request by nitro-marky**
This PR removes Yarn in favour of NPM. It has been tested and Storybook still runs.

Screenshot 2022-12-07 at 10 51 28

This will need to be merged after other PR's as they still contain Yarn. Then there will have to be a follow up PR remove the last bits.

### CODE REVIEW

**Review by n3op2 — DISMISSED** [review]
don't see anything wrong with it for as long as workflows work.

**Review by jonmattgray — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -29,17 +29,17 @@ jobs:
       - name: Cache Node.js modules
         uses: actions/cache@v1
         with:
-          path: ~/.yarn # yarn cache files are stored in `~/.yarn` on Linux/macOS
-          key: ${{ runner.OS }}-node-${{ hashFiles('**/yarn.lock') }}
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-json.json') }}
```
Comment (jonmattgray) [inline .github/workflows/release.yml:33]: `package-lock.json`

Comment (n3op2) [reply .github/workflows/release.yml:33]: well spotted

Comment (nitro-marky) [reply .github/workflows/release.yml:33]: fixed in another pr

**Pull request merged**